### DISCUSSION TITLE
TOLoRS in Ocular

### DISCUSSION THREAD

**Discussion by dinomaster20** [jeffalo/ocular#159] [Ideas]
[Disclaimer: I'm not a contributor to Ocular, just a user]
- - -
My idea is that when you look up something via Ocular that includes code words to one of the rejected suggestions featured in TOLoRS, a box that says exactly what it says on the topic pops up, maybe even with something saying "copy" so you can put that. This would be great, mainly so you don't have to scroll all that way to see if something is rejected. I think this would also work if you put for example: 2.5 - this would bring up 2.5 of TOLoRS. 
- - -
This might or might not only work if you put [category=suggestions] along in there so it knows better, but I'd think it would just pop up at top with any of these ways. This would defiantly help take down rejected suggestions before they got very big, and make them so much easier on the former.

**Comment by jeffalo** [comment]
Interesting. How would this be more useful than visiting TOLoRS directly though?

**Reply by dinomaster20** [reply]
It would save a lot of scrolling and make it easier and much faster to find proof of rejection, especially if you think it's rejected but don't know exactly what you're looking for.
